### PR TITLE
CPed::Avoid and needed changes

### PR DESCRIPTION
**Pull request by erorcun**
I wonder if my style for this function is correct.

### CODE REVIEW

**Comment by aap** [comment]
Check the guidelines. Don't use `unsigned`! I will probably look into CPedType soon because i'm working on CFileLoader.

**Comment by erorcun** [comment]
I took it from DK22 and forgot to change them, sorry. Fixed now.

File: src/entities/Ped.cpp [hunk 1]
```diff
@@ -397,6 +397,60 @@ CPed::OurPedCanSeeThisOne(CEntity* who)
...
+						- (1.25 * (sinValue * someRate)
+							+ GetPosition().x);
+					someDistance = sqrt(someY * someY + someX * someX);
+
+					if (someDistance <= 1.0f && CPed::OurPedCanSeeThisOne((CEntity*)nearestPed)) {
+						m_nPedStateTimer = CTimer::GetTimeInMilliseconds()
+							+ 500 + (m_randomSeed + 3 * CTimer::GetFrameCounter())
+							% 1000 / 5;
+
+						m_fRotationDest += 0.78539819;
```
Comment (Fire-Head) [inline src/entities/Ped.cpp]: m_fRotationDest += DEGTORAD(45.0f);

File: src/entities/Ped.cpp [hunk 2]
```diff
@@ -397,6 +397,60 @@ CPed::OurPedCanSeeThisOne(CEntity* who)
...
+		moveState = m_nMoveState;
+
+		if (moveState != PEDMOVE_NONE && moveState != PEDMOVE_STILL) {
+			nearestPed = m_nearPeds[0];
+
+			if (nearestPed) {
+				if (nearestPed->m_nPedState != PED_DEAD && nearestPed != m_pSeekTarget && nearestPed != m_field_16C
+					&& (CPedType::ms_apPedType[nearestPed->m_nPedType]->m_Type.IntValue
+						& CPedType::ms_apPedType[this->m_nPedType]->m_Avoid.IntValue)) {
+					sinValue = -sin(RADTODEG(m_fRotationCur) * 0.017453216);
```
Comment (Fire-Head) [inline src/entities/Ped.cpp]: 0.017453216 == 1/57.2960307143 == 1/(180/PI) == x / RADTODEG(1)

**Comment by erorcun** [comment]
I was waiting for those comments, thanks. Fixed.

**Pull request merged**